pyccl/tests/test_profiles.py

```diff
@@ -548,6 +551,27 @@ def test_hernquist_cumul2d_accuracy(fourier_analytic):
     assert np.all(res2 < 5E-3)
 
 
+def test_einasto_projected_accuracy():
+    cM = ccl.halos.ConcentrationDuffy08(mass_def='200c')
+    # projected profile from numerical integration
+    p1 = ccl.halos.HaloProfileEinasto(mass_def='200c',
+                                      concentration=cM, truncated=True,
+                                      projected_integrate=True)
+    # FFTLog
+    p2 = ccl.halos.HaloProfileEinasto(mass_def='200c',
+                                      concentration=cM, truncated=True,
+                                      projected_integrate=False)
+
+    M = 1E14
+    a = 0.5
+    rt = np.logspace(-3, 2, 1024)
+    srt1 = p1.projected(COSMO, rt, M, a)[:500]
+    srt2 = p2.projected(COSMO, rt, M, a)[:500]
+
+    res2 = np.fabs(srt2/srt1-1)
+    assert np.all(res2 < 6e-2)
```

Comment:
humm, this is only 6% accurate. Is the limiting factor here FFTLog or quad?

Reply:
I am still testing this part.

Reply:
The limiting factor is the truncated profile given by FFTLog. I set `truncated=False` for FFTLog and they agree to 6e-5.

Reply:
A plot showing the difference between truncated and untruncated `_projected_quad`.
I only plot the range without truncation and thee difference is exactly 0.
![download-1](https://github.com/LSSTDESC/CCL/assets/57552401/5cfd37a3-bb34-46db-afa9-d028408a73d3)